api/stage.go

```diff
@@ -51,6 +51,19 @@ type StageFind struct {
 	PipelineID *int
 }
 
+// StageTasksStatusPatch is the API message for patching task status for all tasks in a stage.
+type StageTasksStatusPatch struct {
```

Comment:
StageAllTaskStatusPatch

Avoid plural form and and use all which is more explicit

Reply:
Done.